src/shared/configUtils.ts

```diff
@@ -44,27 +40,16 @@ export class ConfigUtils {
     return this.#globalConfig;
   }
 
-  public static async getOrCreateIdentityToken(username: string, tokenService: IdentityTokenService): Promise<string> {
-    let identityData = await this.getIdentityData();
-    if (!identityData) {
-      const token = CryptoUtils.generateIdentityToken();
-      const entityId = await tokenService.saveTokenToServer(token);
-      identityData = {
-        identityToken: token,
-        usernameToServerEntityIdMap: {},
-      };
-      identityData.usernameToServerEntityIdMap[username] = entityId;
-      await this.writeIdentityData(identityData);
-      return token;
-    } else {
-      let entityId = identityData.usernameToServerEntityIdMap[username];
-      if (!entityId) {
-        entityId = await tokenService.saveTokenToServer(identityData.identityToken);
-        identityData.usernameToServerEntityIdMap[username] = entityId;
-        await this.writeIdentityData(identityData);
-      }
-      return identityData.identityToken;
```

Comment:
IMO this function didn't belong here. `configUtils` only provides functions to save or fetch values for different configs. But this function was doing extra work (try to fetch, if no there then create a new one, then save it, then return it).

Since this is used during previewing, I refactored it to `previewUtils` instead.

Also simply moved `getIdentityData` up to group it with `writeIdentityData`.